ompi/mca/coll/hcoll/coll_hcoll_component.c

```diff
@@ -251,6 +251,8 @@ static int hcoll_open(void)
 
     cm->libhcoll_initialized = false;
 
+    (void)mca_base_framework_open(&opal_memory_base_framework, 0);
```

Comment:
Why do you want to ignore the return status here -- don't you want to error if the memory hooks fail to get setup properly?